package.json

```diff
@@ -26,78 +26,78 @@
     "test:all": "jest --config=utils/test/jest.config.js"
   },
   "devDependencies": {
-    "@babel/cli": "7.8.3",
-    "@babel/core": "7.8.3",
+    "@babel/cli": "7.8.4",
+    "@babel/core": "7.8.4",
     "@babel/plugin-proposal-class-properties": "7.8.3",
     "@babel/plugin-proposal-object-rest-spread": "7.8.3",
     "@babel/plugin-transform-object-assign": "7.8.3",
     "@babel/polyfill": "7.8.3",
-    "@babel/preset-env": "7.8.3",
+    "@babel/preset-env": "7.8.4",
     "@babel/preset-react": "7.8.3",
-    "@svgr/webpack": "5.0.1",
-    "@testing-library/jest-dom": "4.2.4",
+    "@svgr/webpack": "5.1.0",
+    "@testing-library/jest-dom": "5.1.1",
     "@testing-library/react": "9.4.0",
     "@types/classnames": "2.2.9",
```

Comment:
In order to wrap-off #622, this needs to go too.

Reply:
Removed.